src/tests/cmocka/test_utils.c

```diff
@@ -1955,6 +1955,73 @@ static void test_sss_get_domain_mappings_content(void **state)
      * capaths might not be as expected. */
 }
 
+
+static void test_sss_filter_sanitize_dn(void **state)
+{
+    TALLOC_CTX *tmp_ctx;
+    char *trimmed;
+    int ret;
+    const char *DN = "cn=user,ou=people,dc=example,dc=com";
+
+    tmp_ctx = talloc_new(NULL);
+    assert_non_null(tmp_ctx);
+
+    /* tast that we remove spaces around '=' and ','*/
```

Comment:
Typo here: "tast" instead of "test"

Reply:
good catch, I will fix that and include the comma too.